dev-docs/source/UserSupport.rst

```diff
@@ -205,8 +205,17 @@ For a full release, ``/opt/Mantid/`` is likely the correct install path. Take ca
 
 11. Further diagnosis for process monitoring: `strace <https://strace.io/>`_.
 
+12. To get a stacktrace from C++ follow these steps:
```

Comment:
The bullet points will format more nicely if you bring them forward two spaces, add a new line, and keep each one on one line. Like this:
```
12. To get a stacktrace from C++ follow these steps:

    - Before opening MantidWorkbench, open a terminal and run ``ulimit -c unlimited``
    - Then in the same terminal, open MantidWorkbench with ``/opt/mantidworkbench/bin/mantidworkbench`` and use as normal.
    - If there is a crash, then you will find a ``core.1234`` (some number) file in the current working directory of the terminal, which the user can send to us!
    - Run ``gdb python3 core.1234`` to get the stacktrace from this coredump file.
```

This is alwasy an anoying thing with rst.

Reply:
Ah okay I think you want to move them forward four spaces not, so the hyphen is inline with the the T of 'To get a stacktrace...'. Then it'll indent them without the weird formatting.